### ISSUE TITLE
When changing the mouse sensitivity for the PC, neos does not respect the changes until a computer restart.

### ISSUE THREAD

**Issue by RileyGuy** [Neos-Metaverse/NeosPublic#2544]
## Describe the bug
After changing the mouse sensitivity settings, neos fails to reflect this change until the computer is restarted.

## Relevant issues
N/A

## To Reproduce
Change mouse sensitivity settings.
Neos will not respect this until the computer is restarted.

## Expected behavior
The in-game sensitivity should reflect the changes.

## Bug information (please complete the following information):
 - How often does it happen: Always, until computer is restarted
 - Does the bug persist after restarting Neos? Yes.
 - Neos Version: 2021.6.23.11
 - Neos Platform: Windows, other operating systems were not tested
 - Link to reproduction World/Item: N/A
 - Regression: Not likely

## Reporters:
Myself

**Comment by Frooxius** [comment]
Hmm that's a bit odd. Where in settings are you changing the sensitivity exactly?

If I understand this correctly, you need to restart entire PC, restarting Neos isn't sufficient?

**Comment by RileyGuy** [comment]
Yep to both questions. I'm changing the settings in the mouse control panel - the one under "Additional mouse options" in the windows settings menu.

**Comment by RileyGuy** [comment]
Is it possible we could potentially get an in-game? Or maybe even a launch option since I know the settings menu is kind of messy.

**Comment by Frooxius** [comment]
It sounds like it might be something system related since restarting Neos isn't sufficient. It's a bit odd that the mouse sensitivity in windows actually affects it in Neos, since we're not really using that directly.

Having it adjustable in-game is planned: https://github.com/Neos-Metaverse/NeosPublic/issues/1927

**Comment by RileyGuy** [comment]
Awesome. Yeah, I just figured the neos sensitivity should reflect the system sensitivity somewhat since there's no in-game option.
